src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -70,7 +70,7 @@ private class EmptyEnumStore : EnumStore
 
         // DottedNameNodeSuggestionHandler
         [InlineData("{a:{},b:{},c:{}}.|", "a", "b", "c")]
-        [InlineData("$\"Hello { {a:{},b:{},c:{}}.| } World\"", "a", "b", "c")]
+        [InlineData("$\"Hello { First(Table({a:{},b:{},c:{}})).| } World\"", "a", "b", "c")]
```

Comment:
Can we get a few more suggest tests of the error cases in StrInterpSuggestionHandler?

i.e. `$ |`, `$"foo { |`, ...

Reply:
Added these tests